### PR TITLE
Feature/create circle for each planet

### PR DESCRIPTION
**Pull request by Findsay**

### CODE REVIEW

File: client/build/index.html [hunk 1]
```diff
@@ -5,7 +5,9 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <script src="bundle.js"></script>
-  <title>Document</title>
+  <title>Space Oddity</title>
```
Comment (JamieSK) [inline client/build/index.html:8]: These are going to clash with mine but only because we've done the same thing on different lines.

Comment (Findsay) [reply client/build/index.html:8]: Ahh yeh, no worries though thats an easy conflict fix

**Review by JamieSK — APPROVED** [review]
I like it, though I didn't check your planet colours.

**Review by kulisfunk — APPROVED** [review]
all good to me

**Pull request merged**